README.md

```diff
@@ -1,48 +1,81 @@
-## Einstein Engines
+# Einstein Engines
 
-Einstein Engines is a hard fork of [Space Station 14](https://github.com/space-wizards/space-station-14), built around the ideals and design inspirations of the Baystation family of servers, with a focus on having modular code that anyone can use to make the RP server of their dreams. Our founding organization is based on a democratic system whereby our mutual contributors and downstreams have a say in what code goes into their own upstream. If you are a representative of a former downstream of DeltaV, I would like to invite you to contact us for an opportunity to represent your fork in this new upstream.
+<p align="center"><img src="https://raw.githubusercontent.com/Simple-Station/Einstein-Engines/master/Resources/Textures/Logo/logo.png" width="1080px" /></p>
```

Comment:
This PR looks good so far, I'd probably just replace this with a more generic placeholder logo, at least temporarily until we get a proper one made. I wouldn't want to have Delta-V's logo on the readme file. I'd support merging this as soon as a placeholder is dropped in.